pulsar-metadata/src/main/java/org/apache/pulsar/metadata/api/Stat.java

```diff
@@ -55,4 +62,9 @@ public class Stat {
      * Whether the key-value pair had been created within the current "session".
      */
     final boolean createdBySelf;
+
+    /**
+     * Whether this is the first version of the key-value pair since it has been last created.
+     */
+    final boolean firstVersion;
```

Comment:
Nit: does this always imply created? If so, why not use `boolean created` instead of tying the meaning to the concept of `version`? Based on the usage of this field in the PR, we want to know if this was a creation or a modification event, not if it was the `firstVersion`.

Reply:
`created` has the problem of being misleading on the get operation though. 

If I do a `get()` and receive a value with `created` what does that really mean? The value is created in the sense that it exists :)

Reply:
Thanks for the example, that helps me to understand the name, and I agree that `created` wouldn't make sense in that context.